index.html

```diff
@@ -24,6 +29,8 @@
               <li><a href="#contact">Contact</a></li>
```

Comment:
- [ ] Kindly update the `href` attribute for the menu options, because it's not working, I think that because there's no id called `contact` or `portfolio` etc. Kindly update the ids names.